clusters/app.ci/vault/manifests.yaml

```diff
@@ -46,6 +46,23 @@ spec:
       kind: DockerImage
       name: docker.io/library/vault:1.7.0
 ---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: haproxy
+  namespace: vault
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - name: 2.3.9
+    from:
+      kind: DockerImage
+      # Mirror of docker.io/library/haproxy:2.3.9 because the cluster hit the dockerhub ratelimit
+      name: quay.io/alvaroaleman/haproxy:2.3.9
```

Comment:
Why not import it into the ci registry?

Reply:
That is what it does but the import itself needs to pull it first and that already failed :)